tools/make/packaging.mk

```
@@ -150,6 +150,7 @@ define generate_fpm =
 		--before-remove packaging/$(1)/control/prerm \
 		--config-files /etc/grafana-agent.yaml \
 		--config-files $(ENVIRONMENT_FILE_$(1)) \
+		$(5) \
```

Rather than conditionally passing in extra args, you can always add `--rpm-rpmbuild-define "_build_id_links none"` here, which will be a no-op when building DEBs. 

```suggestion
		--rpm-rpmbuild-define "_build_id_links none" \
```